apps/pushnotif-func/utils/notificationhubServicePartition.ts

```diff
@@ -67,3 +68,59 @@ export const buildNHClient = ({
   AZURE_NH_HUB_NAME,
 }: NotificationHubConfig): NotificationHubsClient =>
   new NotificationHubsClient(AZURE_NH_ENDPOINT, AZURE_NH_HUB_NAME);
+
+export class NotificationHubPartitionFactory {
+  #notificationHubPartitions: DisjoitedNotificationHubPartitionArray;
```

Comment:
```suggestion
  #notificationHubPartitions: DisjointedNotificationHubPartitionArray;
```

Reply:
This is an already defined type, I think it is better to change this type name in a separate PR cause it is used in several files, thanks tho!